openmetadata-service/src/main/java/org/openmetadata/service/events/EventFilter.java

```diff
@@ -70,9 +71,15 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
         .forEach(
             (eventHandler) -> {
               UriInfo uriInfo = requestContext.getUriInfo();
-              if (JwtFilter.EXCLUDED_ENDPOINTS.stream().noneMatch(endpoint -> uriInfo.getPath().contains(endpoint))) {
+              if (EXCLUDED_ENDPOINTS.stream().noneMatch(endpoint -> uriInfo.getPath().contains(endpoint))) {
                 ParallelStreamUtil.runAsync(() -> eventHandler.process(requestContext, responseContext), forkJoinPool);
               }
             });
   }
+
+  private static List<String> getEndpointsFilters() {
+    List<String> filters = new ArrayList<String>(JwtFilter.EXCLUDED_ENDPOINTS);
+    filters.addAll(Arrays.asList("v1/analytics/webAnalyticEvent/collect"));
```

Comment:
nit: in case you don't apply the first suggestion
```suggestion
    List<String> filters = new ArrayList<>(JwtFilter.EXCLUDED_ENDPOINTS);
    filters.add("v1/analytics/webAnalyticEvent/collect");
```